PowerControl/Controller.cs

```diff
@@ -358,5 +413,16 @@ private void osdClose()
             {
             }
         }
+
+        private void DisplayChangesHandler(object? sender, EventArgs e)
+        {
+            if (DeviceManager.RefreshDisplays())
+            {
+                context?.Post((object? state) =>
```

Comment:
Can we just use `
           Dispatcher.CurrentDispatcher.Invoke((_) => rootMenu.Update());`

From `System.Windows.Threading.`.

Reply:
Dispatcher throws an exception

Reply:
I tried at least :) OK, but it appears that we can get rid of this for now.

Reply:
But should we? It still does it's job - fixes multi display

Reply:
It's not a full implementation that would be needed for profiles, but right now it only requires one function - IsSteamDeckOnlyDisplay (temporary name) that would check length of screens array and then if there's only one display check the resolution